cli/cmd/release_create.go

```diff
@@ -184,7 +184,7 @@ Prepared to create release with defaults:
 			if err != nil {
 				return err
 			}
-			if confirmed != "y" {
+			if strings.ToLower(confirmed) == "n" {
```

Comment:
this condition should not be reversed

Reply:
Sure. But why? an `==` is for me more readable vs `!=`

Reply:
I think this was originally written so that the default would be to decline. E.g. "only `yes` will be accepted" and bail in any other case. Sort of a double-check against any bugs in the prompt validation logic.